packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1093,31 +1083,17 @@ export default class NewVm extends BaseComponent {
               </Tooltip>
             )}
           </Item>
-          <Item label={_('vmCpuTopology')}>
-            <select
-              className='form-control'
-              onChange={this._linkState('coresPerSocket')}
-              value={coresPerSocket}
-            >
-              {_('vmChooseCoresPerSocket', message => (
-                <option value=''>{message}</option>
-              ))}
-              {map(this._getCoresPerSocketPossibilities(), coresPerSocket =>
-                _(
-                  'vmCoresPerSocket',
-                  {
-                    nSockets: CPUs / coresPerSocket,
-                    nCores: coresPerSocket,
-                  },
-                  message => (
-                    <option key={coresPerSocket} value={coresPerSocket}>
-                      {message}
-                    </option>
-                  )
-                )
-              )}
-            </select>
-          </Item>
+          {this.props.pool !== undefined && (
```

Comment:
So, not available for `resourceSets`?

Reply:
Yes.

Reply:
Even before, not available for them too?

Reply:
It shows an empty select before because it requires the pool property.

Reply:
IMHO, keep it as before and not hide it for `resourceSets`.

Reply:
But it was a bug, if the select is present, the user should be able to select options. A select without options is not good.

Reply:
If you want you can disabled it and adding tooltip to say that for admin only. 
it was @pdonias 's idea.

Reply:
Done :+1: